test/PostageStamp.test.ts

```diff
@@ -53,1184 +73,1241 @@ describe('PostageStamp', function () {
+          blocksBeforeExpired0 - blocksElapsed3Stamp0 < 0 ? blocksBeforeExpired0 : blocksElapsed3Stamp0;
+        const outpayment3Stamp0 = price0 * blocksCharged3Stamp0 * 2 ** batch.depth;
 
-        await expect(
-          this.postageStamp.copyBatch(
-            stamper,
-            this.batch.initialPaymentPerChunk,
-            this.batch.depth,
-            this.batch.bucketDepth,
-            this.batch.nonce,
-            this.batch.immutable
-          )
-        )
-          .to.emit(this.postageStamp, 'BatchCreated')
-          .withArgs(
-            this.batch.id,
-            this.transferAmount,
-            price + this.expectedNormalisedBalance,
-            stamper,
-            this.batch.depth,
-            this.batch.bucketDepth,
-            this.batch.immutable
-          );
-        const stamp = await this.postageStamp.batches(this.batch.id);
-        expect(stamp[3]).to.equal(price + this.expectedNormalisedBalance);
-      });
+        const blocksElapsed3Stamp1 = (await getBlockNumber()) - buyStamp1Block;
+        const blocksCharged3Stamp1 =
+          blocksBeforeExpired1 - blocksElapsed3Stamp1 < 0 ? blocksBeforeExpired1 : blocksElapsed3Stamp1;
+        const outpayment3Stamp1 = price0 * blocksCharged3Stamp1 * 2 ** batch.depth;
 
-      it('should include pending totalOutpayment in the normalised balance', async function () {
-        const price = 100;
-        await setPrice(price);
+        const blocksElapsed3Stamp2 = (await getBlockNumber()) - buyStamp2Block;
+        const blocksCharged3Stamp2 =
+          blocksBeforeExpired2 - blocksElapsed3Stamp2 < 0 ? blocksBeforeExpired2 : blocksElapsed3Stamp2;
+        const outpayment3Stamp2 = price0 * blocksCharged3Stamp2 * 2 ** batch.depth;
 
-        // mine two blocks, therefore when the next copyBatch happens the totalOutpayment increased 3 times
-        await ethers.provider.send('evm_mine', []);
-        await ethers.provider.send('evm_mine', []);
+        const expectedPot3 = outpayment3Stamp0 + outpayment3Stamp1 + outpayment3Stamp2;
 
-        await expect(
-          this.postageStamp.copyBatch(
-            stamper,
-            this.batch.initialPaymentPerChunk,
-            this.batch.depth,
-            this.batch.bucketDepth,
-            this.batch.nonce,
-            this.batch.immutable
-          )
-        )
-          .to.emit(this.postageStamp, 'BatchCreated')
-          .withArgs(
-            this.batch.id,
-            this.transferAmount,
-            3 * price + this.expectedNormalisedBalance,
-            stamper,
-            this.batch.depth,
-            this.batch.bucketDepth,
-            this.batch.immutable
-          );
-        const stamp = await this.postageStamp.batches(this.batch.id);
-        expect(stamp[3]).to.equal(3 * price + this.expectedNormalisedBalance);
+        expect(await postageStamp.pot()).equal(expectedPot3);
       });
     });
 
+    // describe('when copyBatch creates a batch', function () {
```

Comment:
For what reason are these tests commented/disabled?

Reply:
removed for now

Reply:
tracked here https://github.com/ethersphere/storage-incentives/issues/64

Reply:
we should rewrite these as part of review of bootstrapping procedure